gui_scripts/gui_helpers/general_helpers.py

```diff
@@ -202,3 +202,178 @@ def stop(self):
             self.paused = False
             self.output_hints_signal.emit('Stopping simulation from thread')
         self.pause_condition.wakeOne()
+
+
+class DirectoryTreeView(QtWidgets.QTreeView):
+    """
+    Sets up a new directory tree view
```

Comment:
Same thing here, these always must end with periods which is due to sphinx syntax (see my other comment below).